packages/contentstack-export/src/utils/basic-login.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable max-statements-per-line */
+/* eslint-disable no-console */
+/* eslint-disable no-empty */
+/*!
+ * Contentstack Import
+ * Copyright (c) 2019 Contentstack LLC
+ * MIT Licensed
+ */
+
+import { ExportConfig, ExternalConfig } from '../types';
+import { log } from './logger';
+const {
+  managementSDKClient,
+  isAuthenticated,
+  cliux,
+  configHandler,
+  authHandler,
+} = require('@contentstack/cli-utilities');
+
+const login = async (config: ExternalConfig): Promise<any> => {
+  const client = await managementSDKClient(config);
+  if (config.email && config.password) {
+    const response = await client.login({ email: config.email, password: config.password }).catch(Promise.reject);
```

Comment:
`.catch(Promise.reject)` As per my understanding this reject part is actually doing nothing, correct me if I'm wrong.